incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
```

Comment:
Seems weird to have this in the chart if it's not used by a template. Why not put this in a separate repository and add that as a source in the Chart.yaml?